pkgs/tools/misc/fetch-scm/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchFromGitHub, guile }:
+
+let
+  file = "fetch.scm";
+  pkg = "fetch-scm";
+in
+  stdenv.mkDerivation rec {
+    pname = pkg;
+    version = "v0.1.5";
+
+    src = fetchFromGitHub {
+      owner = "KikyTokamuro";
+      repo = file;
+      rev = version;
+      sha256 = "sha256-H89VCNAYnTwVEqyInATvLHIB7ioe2zvIGTAM2MUo7+g=";
+    };
+
+    dontBuild = true;
+    dontPatchShebangs = true;
+
+    buildInputs = [ guile ];
+
+    installPhase = ''
+      runHook preInstall
+      # strip shebang, because nix removes `--no-auto-compile`
+      tail -n +5 ${file} > ${pkg}
+      # replace with new one
+      shebang="#!${guile.outPath}/bin/guile \\
+      --no-auto-compile -e main -s
+      !#
+      "
+      echo "$shebang$(cat ${pkg})" > ${pkg}
+      install -Dm755 -t $out/bin ${pkg}
```

Comment:
i think we should keep the name of the executable unchanged

Reply:
not the best idea, rather unusual name for an application. also, i maintain package with the same name `fetch-scm` in AUR, it has `fetch-scm` exec

Reply:
I think `fetch.scm` is a fine name, but fair enough